infback.c

```
@@ -361,12 +361,14 @@ int32_t Z_EXPORT PREFIX(inflateBack)(PREFIX3(stream) *strm, in_func in, void *in
 
         case LEN:
             /* use inflate_fast() if we have enough input and output */
-            if (have >= INFLATE_FAST_MIN_HAVE &&
-                left >= INFLATE_FAST_MIN_LEFT) {
+            if (have >= INFLATE_FAST_MIN_HAVE && left >= INFLATE_FAST_MIN_SAFE) {
                 RESTORE();
                 if (state->whave < state->wsize)
                     state->whave = state->wsize - left;
```

Cppcheck (2.20.0)

[style] The function 'put_uint32_msb' is never used.

(unusedFunction)